website/_dogfooding/docs-tests-sidebars.js

```diff
@@ -39,6 +39,12 @@ const sidebars = {
           label: 'External Link test',
           href: 'https://docusaurus.io',
         },
+        {
+          type: 'link',
+          label: 'External Link with description',
+          href: 'https://docusaurus.io',
+          description: 'Some description',
+        },
```

Comment:
I'd suggest creating a brand new category named "Sidebar description tests" or something for all the tests related to description. We'll need more of them